lib/talks/operations/update.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Talks
+  module Operations
+    class Update < Operation
+      include Import[
+        oembed: 'oembed',
+        talk_repo: 'repositories.talk',
+        speaker_repo: 'repositories.speaker',
+        event_repo: 'repositories.event',
+        talks_speakers_repo: 'repositories.talks_speakers'
+      ]
+
+      # 1. Update Talk and Speakers
+      # 2. Create Event if event_id blank otherwise update Event
+      # TODO: add possibility to create new Speakers
+      def call(id, talk_form)
+        oembed = yield generate_oembed(talk_form[:link])
+        talk_repo.transaction do
+          event = yield create_or_update_event(talk_form[:event])
+          talk = yield event ? update_talk(id, talk_form, oembed, event.id) : update_talk(id, talk_form, oembed)
```

Comment:
you can move this logic `event ? update_talk(id, talk_form, oembed, event.id) : update_talk(id, talk_form, oembed)` to method, WDYT?